subgraph/src/Agreement.ts

```diff
@@ -60,7 +64,6 @@ export function handleDisputableAppActivated(event: DisputableAppActivated): voi
   const agreementApp = AgreementContract.bind(event.address)
   const disputableData = agreementApp.getDisputableInfo(event.params.disputable)
   const disputable = loadOrCreateDisputable(event.address, event.params.disputable)
-  disputable.activated = true
```

Comment:
Could you explain the reason for this line being moved to the `loadOrCreateDisputable` function?

Reply:
yeah, for some reason some record on rinkeby was failing given that  the `activated` field was not setted and was required. So we can activate on creation and when is deactivated the other handle set to false